extra/docs_landing.html

```diff
@@ -48,335 +48,178 @@
-      </dd>
-    </dl>
-
-    <h3 id="_store">Store</h3>
-    <dl>
-      <dt id="stack-components-elasticsearch">
-        Elasticsearch
-      </dt>
-      <dd>
-        Elasticsearch is the distributed search and analytics engine at the heart of the Elastic Stack. It provides near real-time search and analytics for all types of data. Whether you have structured or unstructured text, numerical data, or geospatial data, Elasticsearch can efficiently store and index it in a way that supports fast searches. Elasticsearch provides a REST API that enables you to store data in Elasticsearch and retrieve it. The REST API also provides access to Elasticsearch&#8217;s search and analytics capabilities. <a href="en/elasticsearch/reference/current/elasticsearch-intro.html">Learn more about Elasticsearch</a>.
-      </dd>
-    </dl>
-
-    <h3 id="_consume">Consume</h3>
-    <p>Use Kibana to query and visualize the data that&#8217;s stored in Elasticsearch. Or, use the Elasticsearch clients to access data in Elasticsearch directly from common programming languages.</p>
-    <dl>
-      <dt id="stack-components-kibana">
-        Kibana
-      </dt>
-      <dd>
-        Kibana is the tool to harness your Elasticsearch data and to manage the Elastic Stack. Use it to analyze and visualize the data that&#8217;s stored in Elasticsearch. Kibana is also the home for the Elastic Enterprise Search, Elastic Observability and Elastic Security solutions. <a href="en/kibana/current/introduction.html">Learn more about Kibana</a>.
-      </dd>
-      <dt id="stack-components-elasticsearch-clients">
-        Elasticsearch clients
-      </dt>
-      <dd>
-        The clients provide a convenient mechanism to manage API requests and responses to and from Elasticsearch from popular languages such as Java, Ruby, Go, Python, and others. Both official and community contributed clients are available. <a href="en/elasticsearch/client/index.html">Learn more about the Elasticsearch clients</a>.
-      </dd>
-    </dl>
-  </details>
-
-  <h2>Get started</h2>
-  <p>New to Elastic? Learn how you can make the most of your data with the Elastic Stack.
-    Get hands-on with a solution and quickly see data in action, or start from a blank page. </p>
-  <div class="row my-4">
+<div>
+  <p>
+    <a class="inline-block mr-3" href="https://www.elastic.co/guide/index.html#viewall">Browse all docs</a>
+    <a class="inline-block mr-3" href="https://www.elastic.co/guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/new.html">Release docs</a>
+  </p>
+</div>
+<h2 style="font-size:20px;margin-top: 0px;margin-bottom: 0px !important;">Solutions</h2>
```

Comment:
It seems like it *is* possible to mess around with the top <hr> by taking it out of the `convert_document.rb` and `ocbook_compat_spec.rb` and putting it into the `docs_landing.html` file instead. The margins were looking poor with my quick tests, however, so hopefully this can be discussed further as part of the design work next week.